femwell/eme.py

```diff
@@ -256,18 +257,17 @@ def compute_total_S_matrix(meshnames, mesh_info_dict, lengths, wavelength, num_m
     """
     (4) Get a simulatoin a x
     """
-    resolutions = {}
-    resolutions["core"] = {"resolution": 0.02, "distance": 2}
-    resolutions["slab90"] = {"resolution": 0.05, "distance": 2}
-    resolutions["Oxide"] = {"resolution": 0.3, "distance": 2}
+    resolutions = {
+        "core": {"resolution": 0.02, "distance": 2},
+        "slab90": {"resolution": 0.05, "distance": 2},
+        "Oxide": {"resolution": 0.3, "distance": 2},
+    }
 
     ymin = -6 # make sure all objects cross the line
     ymax = 6 # make sure all objects cross the line
 
     meshes = []
     lengths = []
```

Comment:
```suggestion
    lengths = []
    meshnames = []
```
I'd guess this is already prepared for being put in a loop? @simbilod Therefore, I'd suggest not changing this?